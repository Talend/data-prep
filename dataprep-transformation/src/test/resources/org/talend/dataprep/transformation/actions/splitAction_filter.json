{
  "actions": [
    {
      "action": "split",
      "parameters": {
        "column_id": "city",
        "scope": "column",
        "separator": "other",
        "manual_separator": " ",
        "limit":"2",
        "filter": {
          "eq": {
            "field": "state",
            "value": "CA"
          }
        }
      }
    }
  ]
}