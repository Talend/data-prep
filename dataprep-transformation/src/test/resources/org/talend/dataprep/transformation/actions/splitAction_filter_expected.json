{
  "records": [
    {
      "0000": "",
      "0001": "",
      "city": "Santa Fe",
      "state": "NC",
      "tdpId": 1
    },
    {
      "0000": "",
      "0001": "",
      "city": "Santa Fe",
      "state": "NY",
      "tdpId": 2
    },
    {
      "0000": "Santa",
      "0001": "Fe",
      "city": "Santa Fe",
      "state": "CA",
      "tdpId": 3
    },
    {
      "0000": "",
      "0001": "",
      "city": "Grapevine",
      "state": "TX",
      "tdpId": 4
    }
  ]
}